README.md

```diff
@@ -1,72 +1,86 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Todo list App
 
-> Description the project.
+
+> This branch (todo__list) We did it by contributing with my coding partner we did in this branch the followings 👌 :
+
+
+- [x] Setup project with webpack
+- [x] Create index.html using HTML Webpack Plugin
+- [x] Create index.js
+- [x] Add array of to do tasks 💯
+- [x] Iterate array and add dynamically to HTML
+- [x] Create style.css loaded by Webpack Style/CSS Loader
+- [x] Locate files in /src
```

Comment:
- [ ] As the previous reviewer pointed out, this section should contain a description of the project, not what you did on this branch 👀. For example: This app allows users to add tasks, mark them as done, and delete and edit them.